libs/ui/src/pure/MenuBar/styled.ts

```diff
@@ -47,7 +47,7 @@ export const MenuBarWrapper = styled.div<{
   display: flex;
   width: 100%;
   padding: ${({ padding }) => padding || '10px'};
-  z-index: 10;
+  z-index: 9;
   position: sticky;
```

Comment:
The reason for reducing the z-index value for Menubar component is to prevent overlapping of `Menubar` over `Account` or any other modal that need to be rendered in full screen. And I've manually tested with different cases that I can think of and this works well without breaking others around 👍